static/src/javascripts/projects/common/modules/experiments/tests/giraffe.js

```diff
@@ -33,7 +33,8 @@ define([
         this.dataLinkNames = '';
         this.idealOutcome = '';
         this.canRun = function () {
-             return window.guardian.config.page.edition == 'UK';
+             return !window.guardian.config.page.isSensitive &&
```

Comment:
Not sure that it matters, but wondering if `!(isSensitive || isLiveBlog) && edition === 'UK'` - but that's subjective.